pkg/network/transports.go

```diff
@@ -39,6 +42,47 @@ func newAutoTransport(v1 http.RoundTripper, v2 http.RoundTripper) http.RoundTrip
 	})
 }
 
+const (
+	initialTO = float64(50 * time.Millisecond)
+	sleepTO   = float64(30 * time.Millisecond)
+	factor    = 1.4
+	numSteps  = 10
```

Comment:
These read a lot like `wait.Backoff`'s settings. Does it make sense to use that instead and call `backoff.Step()` below everytime a new timeout is needed?

Reply:
Step also does jitter to the actual duration, which I am not sure we want. 
But it all served as inspiration :)

Reply:
Okay, step is not in our dependency, since it's 1.13 :-)
So scratch that.

Reply:
So sad, would've been so pretty 😂